### PR TITLE
docs: (IAC-809) Deployment Operator Documentation Updates

### PR DESCRIPTION
**Pull request by jarpat**
### Changes
Updates to the Deployment Operator Documentation in CONFIG-VARS.md

### CODE REVIEW

**Review by thpang — CHANGES_REQUESTED** [review]

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -43,22 +43,20 @@ Supported configuration variables are listed in the table below.  All variables
...
 
 * In a scenario where you have multiple SAS Viya deployments managed by a single cluster-wide deployment operator, uninstalling one of the SAS Viya deployments does not remove the SAS Viya Deployment Operator from the cluster. However, during the uninstallation workflow, if no SAS Viya deployments that are managed by the cluster-wide deployment operator are detected, the SAS Viya Deployment Operator is also removed.
 
+* If you are running this project using Ansible directly on your workstation, we require Docker to be installed and the executing user should be able to access it. This is required to use the sas-orchestration command. See [ansible usage](user/AnsibleUsage.md#Preparation).
 
-* If you are running this project using Ansible directly on your workstation, we require Docker to be installed and the executing user should be able to access it. This is required to use the Orchestration Tooling CLI see [ansible usage](user/AnsibleUsage.md#Preparation)
+* Using the sas-orchestration command to perform a Viya deployment is only applicable for cadences 2022.12 and beyond. For previous cadences please use the Deployment Operator to perform your deployments.
```
[the remark this answers is not in the document]
Comment (jarpat) [reply docs/CONFIG-VARS.md:59]: That's correct. "Official/Documented" support for using sas-orchestration command to perform a Viya deployment starts in 2022.12.

**Review by riragh — APPROVED** [review]

File: docs/CONFIG-VARS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:59]: Just a suggestion for second sentence to make this requirement more obvious as it is a hard requirement.

 For cadences prior to 2022.12, you must use the Deployment Operator to perform your deployments.

Comment (sayeun) [inline docs/CONFIG-VARS.md:59]: Talked with @supear, we would like the last bullet reads: Using the sas-orchestration deploy command to perform a Viya deployment is only applicable for SAS Viya 2022.12 and later. For previous releases, use the SAS Viya Deployment Operator to perform your deployments.

[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:59]: Susan Pearsall provided input below.

Comment (jarpat) [reply docs/CONFIG-VARS.md]: @sayeun, the bullet point has been updated

**Review by dhoucgitter — APPROVED** [review]

**Review by sayeun — APPROVED** [review]
👍

**Review by thpang — APPROVED** [review]

**Pull request merged**